src/features/events.rs

```diff
@@ -123,6 +142,7 @@ pub async fn check_events_loop(http: Arc<http::Http>) {
             )
             .await
             .expect("unable to send event");
+          message.react(&http_clone, '⌚').await.unwrap();
```

Comment:
I think there should also be some default choice where you don't need to input any time
like for example :one: 10mim :two: 1h :three: 2h :four: 4h :five: 1d